tests/e2e/server/routes.auth.mjs

```diff
@@ -20,12 +20,18 @@ export const baz = {
   canWithdraw: false,
 };
 
-export default function(app) {
+export default function (app) {
   app.post(authPaths.authenticate, wait, async (req, res) => {
     if (!req.body.callbacks) {
```

Comment:
Watch out for this nested if, its getting a bit deep, perhaps it can be flattened somewhat?

Reply:
Ha ha, yeah, this whole route is an absolute mess. I need to spend a whole day refactoring it, but since it's just the mock server, other things are getting priority :) Thanks for bringing this up, I'm going to make a new Jira ticket to do this.